tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml

```diff
@@ -92,6 +92,8 @@ spec:
           echo "ERROR: Expected to fetch sbom for $PYXIS_IMAGES images, but only $SBOM_COUNT were saved"
           exit 1
         fi
+        
+        echo -n "$(workspaces.data.path)/downloaded-sboms" > "$(results.downloadedSbomPath.path)"
```

Comment:
You don't want the workspace in the result. You use this
`INPUT_PATH="$(workspaces.data.path)/$(params.downloadedSbomPath)"`
so this is going to end up evaluating to `/workspace/workspace/downloaded-sboms`  currently, which doesn't exist